android/src/main/java/com/swmansion/reanimated/NodesManager.java

```diff
@@ -390,17 +393,57 @@ public void synchronouslyUpdateUIProps(int viewTag, ReadableMap uiProps) {
   }
 
   public String obtainProp(int viewTag, String propName) {
```

Comment:
After making some research I am afraid that the answer is no, we can't have multiple return types in java function: https://stackoverflow.com/questions/28946291/how-to-return-string-and-int-type-in-a-method. Perhaps we could bypass it, but it would require a lot of extra code 😔